app/notification.go

```diff
@@ -639,6 +721,20 @@ func (a *App) allowChannelMentions(post *model.Post, numProfiles int) bool {
 	return true
 }
 
+// allowGroupMentions returns whether or not the group mentions are allowed for the given post.
+func (a *App) allowGroupMentions(post *model.Post) bool {
+	PERMISSION_USE_GROUP_MENTIONS := model.PERMISSION_USE_CHANNEL_MENTIONS
+	if !a.HasPermissionToChannel(post.UserId, post.ChannelId, PERMISSION_USE_GROUP_MENTIONS) {
```

Comment:
This permission is changed to use the actual `PERMISSION_USE_GROUP_MENTIONS` in the feature branch - I have not done so here because it would cause the build to fail